### ISSUE TITLE
Support on ROS2 client to ROS 1 server

### ISSUE THREAD

**Issue by Briancbn** [ipa-hsd/action_bridge#4]
I am currently trying to use a ROS 2 action client to call the follow_joint_trajectory action server hosted on the Melodic gazebo_ros_control, since it is unavailable in Dashing yet. 

Currently your action bridge only seems to support one direction, which is bridging the ROS 1 client and the ROS2 server. It will be nice to have the reversed direction as well.

**Comment by hsd-dev** [comment]
Sorry for the late response. https://github.com/ipa-hsd/action_bridge/pull/6 takes care of it. I will close the issue once it is merged.
